HW1/P5/P5_wiki_paths.txt

```diff
@@ -0,0 +1,5 @@
+Kevin_Bacon->Time_(magazine)->Harvard_University
+Kevin_Bacon->College_Bowl->Harvard_University
+Kevin_Bacon->Marisa_Silver->Harvard_University
+Kevin_Bacon->John_Lithgow->Harvard_University
+Kevin_Bacon->Six_degrees_of_separation->Harvard_University
```

Comment:
Same paths for Kevin Bacon to Harvard University. But I do not see the reverse paths anywhere.